### ISSUE TITLE
Request - Highlight enrolled tracks on track list

### ISSUE THREAD

**Issue by BNAndras** [exercism/vscode-exercism-cli#7]
The documentation indicates I should be able to pick a new or already started track, but currently, there's no way to distinguish which is which. Perhaps enrolled tracks should appear above ones the user isn't enrolled in?

**Comment by SleeplessByte** [comment]
Yes. The goal will be something like this, which the emacs (non-exercism) integration does.

![image](https://user-images.githubusercontent.com/1964376/192768402-225b2ef5-f7e7-4f9d-9280-de3f6f8d9984.png)

**Comment by niraj8** [comment]
@SleeplessByte Is the screenshot from [canweriotnow/exercism-emacs](https://github.com/canweriotnow/exercism-emacs) emacs package?

**Comment by SleeplessByte** [comment]
Eh. Idk if it's that one. It's one of them 😋
